src/components/calendar/DayBox.tsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import { day as dayI, action, calendarState } from "../../interfaces";
+import Modal from "../page/Modal";
+import { handleEnterPress } from "../../utils/a11y";
+import TaskList from "./TaskList";
+import styled from "styled-components";
+import { backgroundColor, textColor } from "../../theme";
+
+const DayBoxContainer = styled.div`
```

Comment:
This goes for all your components, but making a note here so that we prevent repetition:

Please limit how much custom style you add to your components very carefully. Most cases you have plenty of lines of code only to define styles that apply to a certain component. As a general rule it's better to define a Design System (or main stylesheet) using whatever technology you want (plain CSS, Stylus, SASS, Less, etc.) and use component-specific styles only to adapt the general case to your component's needs. 

This way you separate concerns and make your components more readable altogether. 

Also, because of this implementation road you need to define local, non-reusable components only to create a div with certain attributes, which doesn't make much sense and is a worse practice when compared against the use of classes to apply reusable styles.